ksqldb-rest-model/src/test/java/io/confluent/ksql/rest/model/ImmutabilityTest.java

```diff
@@ -75,6 +75,7 @@ public void shouldBeImmutable() {
         .withKnownImmutableType(KGroupedStream.class)
         .withKnownImmutableType(KGroupedTable.class)
         .withKnownImmutableType(Serde.class)
+        .withKnownImmutableType(Number.class)
```

Comment:
I had to tell the test that we believe `Numbers` to be immutable because otherwise it wouldn't like the fact that we have an`ImmutableMap<String,Number>` to contain the params. Had to change it from `ImmutableMap<String, Object>` because that seemed too generic to make known immutable here. @agavra what do you think?

Reply:
I think we should keep it as `<String, Object>` and instead mark it as `@EffectivelyImmutable` so that it remains generic (in the future, we may have parameters like "structSchemaName" that may not be numbers). You can see that being used (for example) here: https://github.com/confluentinc/ksql/blob/3e48540fbe1a02ed02faa667e424b225f9d7ba20/ksqldb-udf/src/main/java/io/confluent/ksql/types/KsqlStruct.java#L42-L43

Reply:
Done

Reply:
Done!